service/frontend/workflowHandler.go

```diff
@@ -758,19 +780,19 @@ func (wh *WorkflowHandler) RecordActivityTaskHeartbeat(
 		return nil, wh.error(err, scope)
 	}
 
-	domainWrapper := domainWrapper{
```

Comment:
this shadowing got annoying a couple times while figuring out what I wanted the end result to be, so I just got rid of it everywhere.